src/doc/DocAnnotator.js

```diff
@@ -871,21 +871,46 @@ class DocAnnotator extends Annotator {
         }
 
         const lastRange = selection.getRangeAt(selection.rangeCount - 1);
-        const rects = lastRange.getClientRects();
-        if (rects.length === 0) {
-            return;
+        const { endContainer, endOffset } = lastRange;
+        const dummyEl = document.createElement('span');
```

Comment:
Rename this to textEl, selectionEl or something

Reply:
I'll call it `positionEl` since it's used to get position for dialog placement